chroma-manager/tests/framework/utils/install_deps.sh

```diff
@@ -4,5 +4,5 @@ mkdir -p chroma_test_env
 tar -C chroma_test_env -xizf -"
 
 ssh root@$CHROMA_MANAGER "set -ex
-yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/managerforlustre/manager-for-lustre/repo/epel-7/managerforlustre-manager-for-lustre-epel-7.repo
+yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/managerforlustre/manager-for-lustre-devel/repo/epel-7/managerforlustre-manager-for-lustre-devel-epel-7.repo
```

Comment:
Not directly related to this patch, but it seems we should be getting this from [chroma-manager/tests/framework/utils/defaults.sh](https://github.com/intel-hpdd/intel-manager-for-lustre/pull/467/commits/6c0cf3af708c44f8f3d6eb4e02f938f651d54e44#diff-06a3ab104091298eb2e45220cb3e6da4R113)